<launch>
	<node pkg="icp_test" name="velosync" type="velosync_3_fixedout_node" output="screen">
		<rosparam command="load" file="$(find icp_test)/launch/densosync_3.yaml"/>
		<remap from="/combined_cloud" to="/transformed_and_synced"/>
		<remap from="/merged" to="/synced"/>
	</node>
</launch>
